packages/frontend-v2/components/Plan/AddPlanModal.tsx

```diff
@@ -26,18 +27,10 @@ interface AddPlanModalProps {
   setSelectedPlanId: Dispatch<SetStateAction<number | undefined | null>>;
 }
 
-// Mock supported majors till we have scraped and stored majors
-const SUPPORTED_MAJORS = new Map<number, string[]>([
-  [
-    2019,
-    ["Computer Science, BSCS", "Computer Science and Cognitive Psycology, BS"],
-  ],
-  [2020, ["Computer Science, BSCS"]],
-]);
-
 export const AddPlanModal: React.FC<AddPlanModalProps> = ({
   setSelectedPlanId,
 }) => {
+  const { data, error: supportedMajorsError } = useSupportedMajors();
```

Comment:
```suggestion
  const { supportedMajors, error: supportedMajorsError } = useSupportedMajors();
```
I think that the name data here is a little unclear.